Polygon_mesh_processing/doc/Polygon_mesh_processing/Doxyfile.in

```diff
@@ -11,6 +11,7 @@ ALIASES += "cgalNamedParamsBegin=<dl class=\"params\"><dt>Named Parameters</dt><
 ALIASES += "cgalNamedParamsEnd=</table> </dd> </dl>"
 ALIASES += "cgalParamBegin{1}=<tr><td class=\"paramname\">\ref PMP_\1 \"\1\"</td><td>"
 ALIASES += "cgalParamEnd=</td></tr>"
+ALIASES += "cgalDescribePolylineType=A polyline is considered as a set of segments sharing a vertex."
```

Comment:
> a set of segments sharing a vertex

The way I read that, it means that all the segments of the set share the same vertex!

[Somewhere else in CGAL], we can read:
> [a] polyline is a chain of segments, where each two neighboring segments in the chain share a common endpoint

and that it a lot better.

[Somewhere else in CGAL]: https://doc.cgal.org/4.11/Arrangement_on_surface_2/classCGAL_1_1Arr__polyline__traits__2.html